src/services/Analytics/analytics.middleware.ts

```diff
@@ -18,25 +18,40 @@ export const analyticsMiddleware: Middleware<TObject, any, Dispatch<Action>> = (
 ) => (action) => {
   switch (action.type) {
     case addNewAccounts.type: {
-      state.dispatch(
-        trackEvent({
-          name: 'Add Account',
-          params: {
-            qty: action.payload.newAccounts.length,
-            // multiple add accounts are always of the same type and network
-            walletId: action.payload.accountType,
-            networkId: action.payload.networkId
-          }
-        })
-      );
+      // multiple add accounts are always of the same type and network
+      for (let i = 0; i < action.payload.length; i++) {
+        state.dispatch(
+          trackEvent({
+            action: 'Add Account',
+            customDimensions: [
+              {
+                id: 1,
+                value: action.payload[0].wallet
+              },
+              {
+                id: 2,
+                value: action.payload[0].networkId
+              }
+            ]
+          })
+        );
+      }
```

Comment:
@GuillaumeRx loop is used to register every new account as an event (3 added accounts = 3 analytics events), same data is because: [multiple add accounts are always of the same type and network](https://github.com/MyCryptoHQ/MyCrypto/blob/master/src/services/Analytics/analytics.middleware.ts#L26)

Reply:
Sounds good ! (you can simplify by doing a `for (let i in arr)`)